OracleWebLogic/samples/12213-domain-home-in-image/Dockerfile

```diff
@@ -17,7 +17,7 @@
 #
 # Pull base image
 # ---------------
-FROM oracle/weblogic:12.2.1.3-developer
+FROM oracle/weblogic:12213-dev-patch-wls-for-k8s
```

Comment:
Do we require that the customers who use the docker image alone (instead of being managed by the Operator) also have to use the patched WebLogic 12.2.1.3 image?

Reply:
This is sample is specific to patching for WebLogic on Kubernetes and the image needs to be patched no matter if it is built manually or pulled from Docker Store.  When the patched image is available in Docker Store then it will only be required when the image is built manually.